syncstorage/storage/sql/dbconnect.py

```diff
@@ -114,28 +114,22 @@ def __new__(cls, *args, **kwds):
 
 # Common column definitions between BSO and batch upload item tables
 
-def _common_columns(ttl):
-    return (
-        Column("sortindex", Integer),
-        Column("modified", BigInteger, nullable=False),
-        # I'd like to default these to the emptry string and zero,
-        # but MySQL doesn't let you set a default on a TEXT column...
-        Column("payload", PAYLOAD_TYPE, nullable=False),
-        Column("payload_size", Integer, nullable=False),
-        Column("ttl", Integer, server_default=sqltext(str(ttl)))
-    )
```

Comment:
The "bso" and "batch_upload_items" tables actually need different column definitions - in the batch-upload case, we need to know whether any particular field was NULL.